project-board-crew-logic/src/main/java/project/board/crew/logic/structure/user/Users.java

```diff
@@ -0,0 +1,23 @@
+package project.board.crew.logic.structure.user;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+@Entity
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Users {
```

Comment:
Obiekty powinny być w liczbie pojedynczej. Poza tym Usera mamy juz w module User, więc lepiej by było nazwać to jakoś inaczej np member

Reply:
Klasa pomocnicza, nie było jeszcze Usera;